src/Checkbox.jsx

```diff
@@ -96,23 +93,17 @@ export default class Checkbox extends React.Component {
     const state = this.state;
     const prefixCls = props.prefixCls;
     let checked = state.checked;
-    if (typeof checked === 'boolean') {
-      checked = checked ? 1 : 0;
-    }
-    const className = classNames({
-      [props.className]: !!props.className,
-      [prefixCls]: 1,
+
+    const className = classNames([prefixCls], {
+      [props.className]: props.className,
       [`${prefixCls}-checked`]: checked,
-      [`${prefixCls}-checked-${checked}`]: !!checked,
```

Comment:
I don't find the `rc-checkbox-checked-1` or `rc-checkbox-checked-0` class in example or src code.And don't find `ant-checkbox-checked-0` or  `ant-checkbox-checked-1` in antd too.So I delete this.